src/Traits/EwalletTrait.php

```diff
@@ -4,12 +4,15 @@
 namespace EoneoPay\PhpSdk\Traits;
 
 use Symfony\Component\Serializer\Annotation\Groups;
+use Symfony\Component\Validator\Constraints as Assert;
 
 trait EwalletTrait
 {
     /**
      * Ewallet balance.
      *
+     * @Assert\Type(type="string")
```

Comment:
Will this work with an array? Why is it above @var when the others are below?

Reply:
It would not. Is balance a list of `Amount` entities?

No real reason for different order. I'll fix the others in this file to be in alphabetical order

Reply:
Fixed. Balance entity type created.